index.json

```diff
@@ -3,6 +3,56 @@
     "name": "check_run",
     "description": "Check run activity has occurred. The type of activity is specified in the `action` property of the payload object. For more information, see the \"[check runs](/en/rest/reference/checks#runs)\" REST API.",
     "actions": ["completed", "created", "requested_action", "rerequested"],
+    "properties": {
+      "check_run": {
+        "type": "object",
+        "description": "The [check\\_run](/en/rest/reference/checks#get-a-check-run)."
```

Comment:
We will need to turn the relative URLs into absolute URLs

Reply:
That is a bug in the script, since they moved domains, the URLs haven't been changed to absolute URLs on our side

Reply:
Fix on the way in #363